src/replacescopename.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+/**
+ * Normalizes a scoped package name for use as an OS package name.
+ *
+ * @param {?string} [name=''] - the Node package name to normalize
+ * @param {?string} [divider='_'] - the character(s) to replace slashes with
+ */
+module.exports = function replaceScopeName (name, divider) {
+  name = name || ''
+  divider = divider || '_'
```

Comment:
I was researching https://github.com/electron-userland/electron-forge/issues/654, and I found [this](https://fedoraproject.org/wiki/Packaging:Naming#Separators) for RPMs:

> When naming packages for Fedora, the maintainer MUST use the dash '-' as the delimiter for name parts. The maintainer MUST NOT use an underscore '_', a plus '+', or a period '.' as a delimiter.

I think the default needs to change to be a dash.

Reply:
I'll have a PR for this plus a function for the issue I mentioned in the previous comment hopefully sometime tonight.

Reply:
https://github.com/electron-userland/electron-installer-common/pull/10